lms/djangoapps/courseware/tests/test_about.py

```diff
@@ -215,7 +215,7 @@ def setUp(self):
         import_course_from_xml(
             self.store,
             'test_user',
-            TEST_DATA_DIR,
+            TEST_DATA_DIR / u'courses_exports',
```

Comment:
TEST_DATA_DIR is a Path object, not a string.  Slash acts a joiner for path segments.